packages/data/tests/unit/services/elide-metadata-test.ts

```diff
@@ -1,7 +1,7 @@
 import { module, test } from 'qunit';
 import { setupTest } from 'ember-qunit';
 //@ts-ignore
-import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
+import { setupMirage } from 'ember-cli-mirage/test-support';
```

Comment:
I think I remember something about typescript support for newer mirage. Can you try removing the `//@ts-ignore`

Reply:
No out of the box ts support for `ember-cli-mirage/test-support`